Pack3/games/AwShirt/content/AwShirtSlogans.jet

```diff
@@ -1077,7 +1077,7 @@
         },
         {
             "id": 40550,
-            "suggestion": "[KE RALPOLENG] Are we done yet?",
+            "suggestion": "Awnga (lu) hasey mi srak?",
```

Comment:
I feel like it should me "li" instead of "mi" but I don't actually know how such a question would be translated

Reply:
Maybe just "Lu hasey set srak?"